sdks/python/apache_beam/io/gcp/tests/bigquery_matcher_test.py

```diff
@@ -0,0 +1,104 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit test for Bigquery verifier"""
+
+import logging
+import unittest
+
+from hamcrest import assert_that as hc_assert_that
+from mock import Mock, patch
+
+from apache_beam.io.gcp.tests import bigquery_matcher as bq_verifier
+from apache_beam.tests.test_utils import patch_retry
+
+try:
+  from google.cloud import bigquery
+  from google.cloud.exceptions import NotFound
+except ImportError:
+  NotFound = None
```

Comment:
You need to skip tests if you cannot import thing (except for some mock tests maybe). Also it might be clearer to set `bigquery = None` and use that to skip tests.